src/clj/ladders/elo.clj

```diff
@@ -0,0 +1,63 @@
+(ns ladders.elo)
+
+;; q-term name taken from formula on wikipedia page for elo-rating
```

Comment:
these block comments might be more useful as docstrings, they're more repl-friendly that way